.gitlab-ci.py

```diff
@@ -6,13 +6,22 @@
 pipeline.initialize_image("python:3.9-slim")
 
 pipeline.add_children(
-    python.isort(), python.flake8(), python.pytest(), python.evaluate_git_tag_pep404_conformity(),
+    python.isort(),
+    python.flake8(),
+    python.pytest(),
+    python.bdist_wheel(),
     kaniko.execute(
         image_name="thomass/gcip",
         enable_push=(PredefinedVariables.CI_COMMIT_TAG or PredefinedVariables.CI_COMMIT_BRANCH == "main"),
         dockerhub_user_env_var="DOCKER_USER",
         dockerhub_login_env_var="DOCKER_LOGIN",
-    )
+    ),
 )
 
+if PredefinedVariables.CI_COMMIT_TAG:
+    pipeline.add_children(
+        python.evaluate_git_tag_pep404_conformity(),
```

Comment:
Are you sure, checking for pep404 conformity? 
Is PEP 404 semver compliant?

Reply:
> Are you sure, checking for pep404 conformity?

Yes - to be compliant with the mechanics of python packaging tools like `pip`.

> Is PEP 404 semver compliant?

I don't know the exact differences. But the [PEP440 documentation](https://www.python.org/dev/peps/pep-0440/#semantic-versioning) states: _"Semantic versions containing a hyphen (pre-releases - clause 10) or a plus sign (builds - clause 11) are not compatible with this PEP and are not permitted in the public version field."_

Again, to be compliant with the Python ecosystem I would choose PEP440 over SemVer. However in most cases our tags should simply look like `major.minor.micro` - which is basically SemVer.

Reply:
Mhh. I would say, lets go with Semver and forget clause 10 or clause 11. If Semver is the goto Versioning for GCIP, and basically Semver is compliant with PEP440, then lets use only Semver.

Reply:
Final thoughts - because I found regex101 examples in the semver documentation - do we really need the whole feature set of SemVer?
* SemVer https://regex101.com/r/yHqXtC/1
* PEP440 https://regex101.com/r/9xbwXG/1

Reply:
Ok, lets do PEP440. I would recommend to rename the function.
```suggestion
        python.evaluate_git_tag_pep440_conformity(),
```